maps/away/noctis/noctis_shuttle.dm

```diff
@@ -0,0 +1,30 @@
+/datum/shuttle/autodock/overmap/raptor
```

Comment:
I can add a docking port for this on Nerva once it's merged, or add landmarks around Nerva for it.

Reply:
We should already have generic landmarks around Nerva that are accessible to all shuttles that fit. The shuttle's a pretty strange size, but I imagine you could probably get it to match one of the docking ports.